linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/configure_opensc_card_drivers/bash/shared.sh

```diff
@@ -0,0 +1,14 @@
+# platform = Red Hat Enterprise Linux 7,multi_platform_fedora
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+
+. /usr/share/scap-security-guide/remediation_functions
+populate var_smartcard_drivers
+
+grep -qs "card_drivers =" /etc/opensc*.conf && \
+	sed -i "s/card_drivers =.*/card_drivers = $var_smartcard_drivers;/g" /etc/opensc*.conf
+if ! [ $? -eq 0 ]; then
+	sed -i "s/.*card_drivers =.*/        card_drivers = $var_smartcard_drivers;/g" /etc/opensc*.conf
+fi
```

Comment:
Will `replace_or_append` function work here?

Reply:
No because of the uniqueness of `opensc.conf` naming and how the `.conf` is structured and formatted as well.